### PR TITLE
fix: Run addons only when needed

### PR DESCRIPTION
**Pull request by lolgab**

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=1.3.2
+sbt.version=1.3.4
```
Comment (josemiguelmelo) [inline project/build.properties]: Can't you bump to 1.3.5?

Comment (lolgab) [reply project/build.properties]: ✅ Done

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 version: 2.1
 
 orbs:
-  codacy: codacy/base@0.3.12
-  codacy_plugins_test: codacy/plugins-test@0.6.0
+  codacy: codacy/base@1.2.0
```
Comment (josemiguelmelo) [inline .circleci/config.yml]: Bump to 1.2.1

Comment (lolgab) [reply .circleci/config.yml]: ✅Done

**Pull request merged**